src/internal/storage/track/postgres_tracker.go

```diff
@@ -214,17 +253,21 @@ func SetupPostgresTrackerV0(ctx context.Context, tx *sqlx.Tx) error {
 }
 
 var schema = `
-	CREATE TABLE IF NOT EXISTS storage.tracker_objects (
+	CREATE TABLE storage.tracker_objects (
 		int_id BIGSERIAL PRIMARY KEY,
 		str_id VARCHAR(4096) UNIQUE,
-		tombstone BOOLEAN NOT NULL DEFAULT FALSE,
 		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
 		expires_at TIMESTAMP
 	);
 
-	CREATE TABLE IF NOT EXISTS storage.tracker_refs (
+	CREATE TABLE storage.tracker_refs (
 		from_id INT8 NOT NULL,
 		to_id INT8 NOT NULL,
 		PRIMARY KEY (from_id, to_id)
-	);	
+	);
+
+	CREATE INDEX ON storage.tracker_refs (
```

Comment:
Have you thought about the performance implication of having the multicolumn index be `to_id, from_id` rather than `from_id, to_id`? It seems like it should be the latter given that the primary key has the same structure.